sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResult.ts

```diff
@@ -188,21 +188,22 @@ function convertGeneratedSentenceSentiment(
     confidenceScores: sentence.confidenceScores,
     sentiment: sentence.sentiment,
     text: sentence.text,
-    minedOpinions: sentence.aspects?.map(
-      (aspect: SentenceAspect): MinedOpinion => ({
-        aspect: {
-          confidenceScores: {
-            ...(aspect.confidenceScores as SentimentConfidenceScores),
-            neutral: 0
+    minedOpinions: sentence.aspects ?
```

Comment:
Does accessing `aspects` throw if there are no `aspects`? (i.e. with API version v3.0, there is no `aspects` attribute, so accessing an attribute that does not exist throws)

Reply:
It will be an empty list if aspects aren't returned. This SDK can only be used with 3.1-preview.1, since we don't support multi API yet.